x/concentrated-liquidity/math/math.go

```diff
@@ -44,6 +46,10 @@ func Liquidity1(amount sdk.Int, sqrtPriceA, sqrtPriceB sdk.Dec) sdk.Dec {
 	sqrtPriceBBigDec := osmomath.BigDecFromSDKDec(sqrtPriceB)
 
 	diff := sqrtPriceBBigDec.Sub(sqrtPriceABigDec)
+	if diff.Equal(osmomath.ZeroDec()) {
+		panic(fmt.Sprintf("diff is zero: sqrtPriceA %s sqrtPriceB %s", sqrtPriceA, sqrtPriceB))
+	}
```

Comment:
Note: makes the error more descriptive. Since this should never happen in-practice, I think we should keep this as is instead of returning error

Reply:
Agreed, can we address it in this PR or make an issue so that we make sure we come back to it?